lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -234,6 +239,16 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
     {{:ok, spec: spec}, %{state | log_metadata: log_metadata}}
   end
 
+  @impl true
+  def handle_notification({:estimation, estimations}, {:track_sender, track_id}, _ctx, state) do
+    notification = %TrackNotification{
+      track_id: track_id,
+      notification: bitrate_notification(estimations)
+    }
+
+    {{:ok, notify: {:publish, notification}}, state}
```

Comment:
The API should more look like `{:publish, topic, msg}` where `topic` is track_id.

Reply:
We discussed it and I think we came to an agreement that this is good enough for now and we will be overhauling the API at a later stage